pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -98,11 +98,16 @@ func (p RESTStorageProvider) storage(version schema.GroupVersion, apiResourceCon
 			clusterRolesStorage = clusterrolestore.NewREST(restOptionsGetter)
 			clusterRoleBindingsStorage = clusterrolebindingstore.NewREST(restOptionsGetter)
 
+			rolesStandardStorage, _ := rolesStorage.(rest.StandardStorage)
```

Comment:
why is the cast necessary? the objects returned from NewREST should satisfy the interface